pytket/docs/extensions.md

```diff
@@ -0,0 +1,140 @@
+# Extension packages
+
+The pytket extensions are separate python modules which allow pytket to interface with backends from a range of providers including quantum devices from Quantinuum and IBM.
+In pytket a `Backend` represents a connection to a QPU (Quantum Processing Unit) or simulator for processing quantum circuits. One can also access additional quantum devices and simulators via the cloud through the extensions for [Braket](inv:pytket-braket:std:doc#index) .
+
+Additionally, the extensions allow pytket to cross-compile circuits from different quantum computing libraries with the extensions for [qiskit](inv:pytket-qiskit:std:doc#index), [cirq](inv:pytket-cirq:std:doc#index) and [pennylane](inv:pytket-pennylane:std:doc#index) . This enables pytket's compilation features to be used in conjunction with other software tools.
+
+The additional modules can be installed adding the extension name to the installation command for pytket. For example pytket-quantinuum can be installed by running
+
+```
+pip install pytket-quantinuum
+```
+
+The types of `Backend` available in pytket are the following
+
+## Types of Backend
+
+- **QPUs** - These are real quantum computers that return shots based results. E.g the [QuantinuumBackend](inv:#*.QuantinuumBackend) .
+- **Cloud Access** - Cloud backends allow pytket to interface with cloud platforms to access additional QPUs and simulators. E.g [BraketBackend](inv:#*.BraketBackend) .
```

Comment:
Updated some of these links to use robust intersphinx linking. Will do the rest in another PR.